modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/page.jsp

```diff
@@ -36,27 +36,30 @@
 			creationMenu: <%= jsonSerializer.serializeDeep(creationMenu) %>,
 			currentURL: '<%= PortalUtil.getCurrentURL(request) %>',
 			dataProviderKey: '<%= dataProviderKey %>',
-			formId: '<%= GetterUtil.getString(formId) %>',
+			formId: '<%= GetterUtil.getString(formId) %>' || undefined,
```

Comment:
I think a more common pattern we've used in the past is to wrap this in a `<c:if>` so that a line is not rendered at all. This renders some ugly looking markup with line jumps in it which is something I'm not particularly fond of, but those `|| undefined` also make me uneasy somehow 😂

Reply:
Sorting looks great, though! ❤️